bitcoin/tests/psbt.rs

```diff
@@ -26,10 +26,10 @@ macro_rules! hex_script {
     };
 }
 
-macro_rules! hex_psbt {
-    ($s:expr) => {
-        Psbt::deserialize(&<Vec<u8> as FromHex>::from_hex($s).unwrap())
-    };
+#[track_caller]
+pub fn hex_psbt(s: &str) -> Psbt {
```

Comment:
Agree with @tcharding that this doesn't need to public.

Reply:
thanks, good catch.

Reply:
done